local_planner/src/nodes/common.cpp

```diff
@@ -13,26 +13,14 @@ float distance2DPolar(int e1, int z1, int e2, int z2) {
   return sqrt(pow((e1 - e2), 2) + pow((z1 - z2), 2));
 }
 
-float computeL2Dist(const geometry_msgs::Point& position,
-                    const pcl::PointXYZ& xyz) {
-  return sqrt(pow(position.x - xyz.x, 2) + pow(position.y - xyz.y, 2) +
-              pow(position.z - xyz.z, 2));
-}
-
-float distance3DCartesian(const geometry_msgs::Point& a,
-                          const geometry_msgs::Point& b) {
-  return sqrt((a.x - b.x) * (a.x - b.x) + (a.y - b.y) * (a.y - b.y) +
-              (a.z - b.z) * (a.z - b.z));
-}
-
 // transform polar coordinates into Cartesian coordinates
```

Comment:
I think we should make a big note here on the units. The fact that `e` and `z` are integers (which I find questionable in the first place in this context), and represent elevation and orientation in degrees should be documented somewhere, no?

Reply:
Documentation would be much appreciated!